### ISSUE TITLE
Documentation in the Auth setup is broken.

### ISSUE THREAD

**Issue by Price775** [google/google-api-javascript-client#468]
function init() {
  gapi.load('auth2', function() { // Ready. });
}

should be

function init() {
  gapi.load('auth2', function() { /* Ready.*/ });
}

**Comment by robotdan** [comment]
This error is on the [Google Sign-In JavaScript client reference](https://developers.google.com/identity/sign-in/web/reference#gapiauth2initwzxhzdk19paramswzxhzdk20) page in the `Auth Setup` section.

Ideally the example would demonstrate what to do in the `Ready` block. 

```javascript
function init() {
  gapi.load('auth2', function() {
    /* Ready. Make a call to gapi.auth2.init or some other API */
  });
}
```

**Comment by grant** [comment]
Thanks.
This will be fixed soon with internal change 253834992.

**Comment by grant** [comment]
Fixed. Changes are live!
